v22.2/copy-from.md

```diff
@@ -104,25 +104,25 @@ SELECT * FROM users WHERE id IN ('8a3d70a3-d70a-4000-8000-00000000001d', '9eb851
 (2 rows)
 ~~~
 
-### Copy CSV delimited data
+### Copy CSV-delimited data to CockroachDB
 
 You can copy CSV data into CockroachDB using the following methods:
 
-- [Copy CSV delimited data from stdin](#copy-csv-delimited-data-from-stdin)
-- [Copy CSV delimited data from stdin with an escape character](#copy-csv-delimited-data-from-stdin-with-an-escape-character)
-- [Copy CSV delimited data from stdin with a header](#copy-csv-delimited-data-from-stdin-with-a-header)
-- [Copy CSV delimited data from stdin with hex encoded byte array data](#copy-csv-delimited-data-from-stdin-with-hex-encoded-byte-array-data)
+- [Copy CSV-delimited data from `stdin`](#copy-csv-delimited-data-from-stdin)
+- [Copy CSV-delimited data from `stdin` with an escape character](#copy-csv-delimited-data-from-stdin-with-an-escape-character)
+- [Copy CSV-delimited data from `stdin` with a header](#copy-csv-delimited-data-from-stdin-with-a-header)
+- [Copy CSV-delimited data from `stdin` with hex-encoded byte array data](#copy-csv-delimited-data-from-stdin-with-hex-encoded-byte-array-data)
 
-#### Copy CSV delimited data from stdin
+#### Copy CSV-delimited data from `stdin`
 
-Run the following SQL statement to create a new table that you will load with CSV formatted data:
+First, create a new table that you will load with CSV-formatted data:
```

Comment:
I like your change tbh. But this was recently added to the style guide: https://github.com/cockroachdb/docs/blob/master/StyleGuide.md#use-ordered-lists-when-there-are-multiple-steps-in-a-section

Reply:
Ah, I knew there was something here. Fixed.